js/repl/src/registerServiceWorker.js

```diff
@@ -0,0 +1,108 @@
+// In production, we register a service worker to serve assets from local cache.
```

Comment:
This seems like overkill given most of the assets come from a CDN. We can keep it if you feel it provides value though. What do you think?